.markdownlint.yaml

```diff
@@ -67,15 +67,15 @@ MD013:
   # Number of characters
   line_length: 80
   # Number of characters for headings
-  heading_line_length: 80
+  # heading_line_length: 80
```

Comment:
Is it necessary? I think it still makes sense to have shorter headings whenever possible, and keeping such a rule may give extra motivation to be concise.

In rare cases, we can ignore checks in specific lines, rather than disabling rules entirely: https://github.com/DavidAnson/markdownlint#configuration